exec/java-exec/src/main/java/org/apache/drill/exec/store/ischema/InfoSchemaFilterBuilder.java

```diff
@@ -73,9 +73,12 @@ public ExprNode visitFunctionCall(FunctionCall call, Void value) throws RuntimeE
       case "like": {
         ExprNode arg0 = call.args.get(0).accept(this, value);
         ExprNode arg1 = call.args.get(1).accept(this, value);
+        ExprNode arg2 = call.args.size() > 2 ? call.args.get(2).accept(this, value) : null;
```

Comment:
do you want to update these variables to have meaningful names about what each of the arguments will be used for like you did above? Would make it a little clearer why the ternary is only needed in the last case because the escape is the only optional parameter.